pkg/cmd/provisioning/ibmcloud/create.go

```diff
@@ -0,0 +1,177 @@
+package ibmcloud
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+
+	credreqv1 "github.com/openshift/cloud-credential-operator/pkg/apis/cloudcredential/v1"
+	"github.com/openshift/cloud-credential-operator/pkg/cmd/provisioning"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/util/yaml"
+)
+
+const (
+	secretManifestsTemplate = `apiVersion: v1
+stringData:
+  ibmcloud_api_key: %s
+kind: Secret
+metadata:
+  name: %s
+  namespace: %s
+type: Opaque`
+
+	manifestsDirName = "manifests"
+)
+
+var (
+	// CreateOpts captures the options that affect creation of the generated
+	// objects.
+	CreateOpts = options{
+		TargetDir: "",
+	}
+)
+
+// NewCreateCmd implements the "create" command for the credentials provisioning
+func NewCreateCmd() *cobra.Command {
+	createCmd := &cobra.Command{
+		Use:              "create",
```

Comment:
by command conventions that we follow, this should be `create-secrets` I feel. @dgoodwin WDYT?

Reply:
That's no problem. Changed to `create-secrets`